features/attached_commands.feature

```diff
@@ -293,30 +293,44 @@ Feature: Command behaviour when attached to an UA subscription
         https://esm.ubuntu.com/ubuntu/ trusty-infra-updates/main amd64 Packages
         """
 
+    @wip
     @series.all
     Scenario Outline: Help command on an attached machine
         Given a `<release>` machine with ubuntu-advantage-tools installed
         When I attach `contract_token` with sudo
         And I run `ua help esm-infra` with sudo
         Then I will see the following on stdout:
             """
-            name: esm-infra
-            entitled: yes
-            status: enabled
-            help:
-            esm-infra help
-            Information about esm-infra
+            Name:
+            esm-infra
+
+            Entitled:
+            yes
+
+            Status:
+            enabled
+
+            Help:
+            UA Infra: Extended Security Maintenance is enabled by default on entitled
+            workloads. It provides access to a private PPA which includes available
+            high and critical CVE fixes for Ubuntu LTS packages in the Ubuntu Main
+            repository between the end of the standard (free) Ubuntu LTS security
+            maintenance and its end of life. You can find out more about the esm
+            service at https://ubuntu.com/security/esm.
             """
         When I run `ua help esm-infra --format json` with sudo
         Then I will see the following on stdout:
             """
-            {"name": "esm-infra", "entitled": "yes", "status": "enabled", "help": "esm-infra help\nInformation about esm-infra"}
+            {"name": "esm-infra", "entitled": "yes", "status": "enabled", "help": "UA Infra: Extended Security Maintenance is enabled by default on entitled\nworkloads. It provides access to a private PPA which includes available\nhigh and critical CVE fixes for Ubuntu LTS packages in the Ubuntu Main\nrepository between the end of the standard (free) Ubuntu LTS security\nmaintenance and its end of life. You can find out more about the esm\nservice at https://ubuntu.com/security/esm.\n"}
             """
         When I run `ua help invalid-service` with sudo
         Then I will see the following on stdout:
             """
-            name: invalid-service
-            help: No help available for service "invalid-service"
+            Name:
```

Comment:
Is this really desired output versus a usage error?


```
usage: ua help [service_name] [flags]
Unknown service_name 'a'.
```

Reply:
I would take service out of the wording. Just "no help available for "user-input"